modules/samples/oauth2/playground2/src/main/webapp/oauth2.jsp

```diff
@@ -480,7 +538,60 @@
                 </form>
             </div>
             <%} %>
-
+            <% } else if (implicit_response_type != null &&
+                    OpenIDConnectConstants.ID_TOKEN.equals(implicit_response_type)) {
+            %>
+            <div>
+                <table class="user_pass_table">
+                    <tbody>
+                    <tr>
+                        <td><h5>ID Token:</h5></td>
+                        <td><input id="idToken" name="idToken" style="width:800px"/>
+                            <script type="text/javascript">
+                                document.getElementById("idToken").value = getIDtoken();
+                            </script>
+                        </td>
+                        <td>
+                        </td>
+                    </tr>
+                    <tr>
+                        <script type="text/javascript">
+                            var decodedIdToken = JSON.parse(getDecodedIDToken());
+                            makeList(decodedIdToken);
+                        </script>
+                    </tr>
+                    </tbody>
+                </table>
+            </div>
+            <% } else if (implicit_response_type != null &&
```

Comment:
Can't we simplify 
`implicit_response_type != null && OpenIDConnectConstants.ID_TOKEN_TOKEN.equals(implicit_response_type)
`to
`OpenIDConnectConstants.ID_TOKEN_TOKEN.equals(implicit_response_type)`